examples/usdConverter/README.md

```diff
@@ -0,0 +1,30 @@
+# Converting between SDF and USD
+
+This example shows how a world in a SDF file can be converted to [USD](https://graphics.pixar.com/usd/release/index.html).
+
+## Requirements
+
+You will need all of the dependencies for sdformat, along with the following additional dependencies:
+* USD: [installation instructions](https://github.com/PixarAnimationStudios/USD/blob/release/README.md#getting-and-building-the-code)
+* [ignition-common4](https://github.com/ignitionrobotics/ign-common)
+
+## Setup
+
+Build sdformat, and then run the following commands to build the example (run these commands from this example directory):
+```bash
+mkdir build
+cd build
+cmake ..
+make
+```
+
+You should now have an executable named `usdConverter`, which can be used to convert a SDF world file to a USD file.
+The following command converts the example `shapes.sdf` file to its USD representation, stored in a file called `shapes.usda` (run this command from the `build` directory):
+```bash
+./usdConverter ../shapes.sdf shapes.usda
```

Comment:
right now, this is not working - whenever I try to run the executable generated by the example, I get the following error:
```
./usdConverter: error while loading shared libraries: libusd_usdPhysics.so: cannot open shared object file: No such file or directory
```

This was not happening with this executable before turning all of this code into a separate component, so perhaps the component is not being created correctly at the moment. Does anyone know what's happening here? cc @scpeters